docker/multi-process/Dockerfile

```diff
@@ -1,32 +1,24 @@
 FROM ubuntu:14.04
 
-ADD docker/scripts/prepare /scripts/prepare
+COPY docker/scripts/prepare /scripts/
 RUN /scripts/prepare
 
-ADD docker/multi-process/scripts/standalone-packages /scripts/standalone-packages
+COPY docker/multi-process/scripts/standalone-packages /scripts/
 RUN /scripts/standalone-packages
 
 WORKDIR /app
-
-ADD ["Gemfile", "Gemfile.lock", "/app/"]
-ADD lib/gemfile_helper.rb /app/lib/
-ADD vendor/gems /app/vendor/gems
-
-RUN chown -R huginn:huginn /app && \
-    sudo -u huginn -H echo "gem 'sqlite3', '~> 1.3.11'" >> /app/Gemfile && \
-    sudo -u huginn -H LC_ALL=en_US.UTF-8 RAILS_ENV=production ON_HEROKU=true bundle install --without test development --path vendor/bundle -j 4
-COPY . /app
+COPY ./ /app/
 
 ARG OUTDATED_DOCKER_IMAGE_NAMESPACE=false
 ENV OUTDATED_DOCKER_IMAGE_NAMESPACE ${OUTDATED_DOCKER_IMAGE_NAMESPACE}
 
-ADD ["docker/scripts/setup", "docker/scripts/setup_env", "docker/multi-process/scripts/init", "/scripts/"]
-
+COPY docker/scripts/setup /scripts/
 RUN /scripts/setup
 
-VOLUME /var/lib/mysql
-
 EXPOSE 3000
 
+COPY docker/multi-process/scripts/init /scripts/
 CMD ["/scripts/init"]
 
+RUN chgrp -R 0 /app && chmod -R g=u /etc/passwd /app
```

Comment:
Can this be done in `scripts/init` or `scripts/setup`?

Reply:
Good idea, I'll move them to `scripts/setup`.